src/redux/auth-reducer.js

```diff
@@ -23,55 +27,89 @@ function setLoginError(loginError) {
   }
 }
 
-function callLoginApi(email, password, callback) {
-    setTimeout(() => {
-      if (email === 'admin@example.com' && password === 'admin') {
-        return callback(null);
+function setUserPath(userPath) {
+  return {
+    type: SET_USER_PATH,
+    userPath
+  };
+}
+
+function callLoginApi(ID, password, callback) {
+  // setTimeout(() => {
+  //   if (email === 'admin@example.com' && password === 'admin') {
+  //     return callback(null);
+  //   } else {
+  //     return callback(new Error('Invalid email and password'));
+  //   }
+  // }, 1000);
```

Comment:
Will we need this commented out setTimeout function now that the actual login logic has been implemented?

Reply:
Nope, that can get tossed out too